addon/components/power-select.js

```diff
@@ -512,7 +512,7 @@ export default Component.extend({
   },
 
   _updateOptionsAndResults(opts) {
-    if (get(this, 'isDestroyed')) {
+    if (get(this, 'isDestroying')) {
```

Comment:
`this.isDestroyed || this.isDestroying` should be sufficient and you should not need to go through Ember.get for either prop.